tests/baselines/reference/findAllRefsImportMeta.baseline.jsonc

```diff
@@ -0,0 +1 @@
+[]
```

Comment:
Ooops, it's empty because `definition` is removed. Currently `definition` is required. It might break vscode if we make it optional.

https://github.com/microsoft/TypeScript/blob/733eff2b104540648dfe0c661f08b019165bdd09/src/services/types.ts#L1042-L1045